tests/FSharp.Compiler.ComponentTests/ErrorMessages/SuggestionsTests.fs

```diff
@@ -129,22 +129,22 @@ module Collections =
 
     let f () = printfn "%s" "Hello"
 
-open Collectons
+open Collections_
```

Comment:
If possible, keep the code here, before it was clearer what was going on here.

Reply:
An alternative corruption would be `Collect_ions`, but, sure. I'll drop.